libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -213,7 +213,27 @@ func (m *Manager) GetPids() ([]int, error) {
 		return nil, err
 	}
 
-	return cgroups.ReadProcsFile(dir)
+	var pids []int
+	// collect pids from all sub-cgroups
+	err = filepath.Walk(dir, func(p string, info os.FileInfo, iErr error) error {
+		if info.IsDir() {
+			return nil
+		}
+
+		dir, file := filepath.Split(p)
+		if file == "cgroup.procs" {
+			if iErr != nil {
```

Comment:
I don't want to stop process on errors for other files. In user-namespace this can be an issue.